app/assets/javascripts/no_sunet.js

```diff
@@ -0,0 +1,43 @@
+$(document).on('click', '#no-sunet-button', function(){
+  setTimeout(function(){
```

Comment:
Isn't that what's already happening from here: https://github.com/sul-dlss/sul-requests/blob/master/app/assets/javascripts/toggle_show_tabs.js ?